### ISSUE TITLE
Issues connecting to GTalk?

### ISSUE THREAD

**Issue by OmegaZero** [ChatSecure/ChatSecure-iOS#974]
I signed on with the following

Username: myemail@gmail.com
Password: &lt;my google password&gt;
Hostname: blank
Port: left alone

I get [all sorts of warnings and errors](https://imgur.com/a/5wBep) but the connection still appears to work - is there something I'm doing wrong?  I couldn't seem to find a "how to" on connecting to GTalk...

**Comment by zuglufttier** [comment]
You're getting those warnings/errors because Google does not care about XMPP anymore. It's still working somehow but you should move on and use a decent server. Here is a starting point: https://conversations.im/compliance/

**Comment by OmegaZero** [comment]
The problem is everyone I talk to uses GTalk - would I be able to use a different server and still sign in through Google?

**Comment by chrisballinger** [comment]
Google Talk is deprecated and will be removed in a future release, it has
been completely neglected by Google for years. If you want to chat with
Google people you should use Hangouts instead.

**Comment by OmegaZero** [comment]
Problem with hangouts is it doesn't support OTR or OMEMO (at least, that I know of) so don't know if that solves my problem.  Again I ask, is there a different server that DOES support XMPP and OMEMO that we can use a google login?  Or do we just need to switch servers and make new accounts.

**Comment by vitalyster** [comment]
@OmegaZero you can set up Hangouts gateway on your XMPP server and get Google contacts in your XMPP contact list.

**Comment by OmegaZero** [comment]
@vitalyster Yes but if they are using Hangouts still would it support OMEMO?

**Comment by vitalyster** [comment]
@OmegaZero OTR by its nature is usable over any network, but they need to use OTR-compatible client and not the Google Hangouts official clients. If you want to talk with group of tech "geeks" sure you all can switch to modern XMPP server(also with OMEMO) and ChatSecure, but it's doubt you can make "normies" interested in this.

**Comment by OmegaZero** [comment]
Right, we just use pidgin with OTR over XMPP with a google login right now, not GTalk or Hangouts specifically.  OK so basically we HAVE to switch servers.  Is there a recommended server that I can get my tech friends to switch to but will also let me talk to people that are still left on GTalk/Google's XMPP server (albeit without the advances of the modern XMPP server in chats with them)?

Thanks for the info so far!

**Comment by vitalyster** [comment]
@OmegaZero I'm not sure about its state, but ChatSecure team provided its own zom.im server and zom messenger (branded ChatSecure)

**Comment by vitalyster** [comment]
For the second part of question: first you can just install [purple-hangouts](https://bitbucket.org/EionRobb/purple-hangouts) plugin into your pidgin. It can be tricky to register, but it mostly works and provide much more "casual" features than deprecated gtalk connection.
The same plugin (and many other pidgin plugins) possible to run on server side as "xmpp transport" with help of [Spectrum IM Transports](https://spectrum.im/) and it will allow you connect from multiple xmpp clients (and ChatSecure!) and got the same contact list as in pidgin on desktop. However, I'm not sure about public xmpp servers with hangouts gateway enabled, and you may need to set up it yourself
